src/components/AdvancedSelect.vue

```diff
@@ -95,6 +95,11 @@
               href="#" @click="select($event, option.value)">
               <i v-if="option.icon" :class="['fa', 'pos-rel', option.icon]"/>
               {{option.text}}
+              <small
+                v-if="option.subtext"
+                class="text-muted">
+                &mdash; {{option.subtext}}
```

Comment:
I would leave the dash inside the subtext value.

Reply:
I will update the dash and wrap it in a span so it can by hidden if not needed. Thanks!